torch/csrc/utils/tensor_new.cpp

```diff
@@ -21,11 +21,9 @@ using namespace at;
 namespace torch { namespace utils {
 
 static void maybe_initialize_cuda(const at::Type &type) {
-#ifdef WITH_CUDA
   if (type.is_cuda()) {
     torch::cuda::lazy_init();
```

Comment:
err, we should change the namespace/function name to match the file.  Instead of torch::cuda::lazy_init, torch::utils::cuda_lazy_init.  Also maybe add a comment to the .h file indicating why it's not in /csrc/cuda (because we want to unconditionally compile it).